packages/react/src/icons.scss

```diff
@@ -1 +1,26 @@
 @import "@fontsource/material-icons";
+
+.material-icons {
+  font-family: 'Material Icons';
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px;  /* Preferred icon size */
+  display: inline-block;
+  line-height: 1;
+  text-transform: none;
+  letter-spacing: normal;
+  word-wrap: normal;
+  white-space: nowrap;
+  direction: ltr;
+
+  /* Support for all WebKit browsers. */
+  -webkit-font-smoothing: antialiased;
+  /* Support for Safari and Chrome. */
+  text-rendering: optimizeLegibility;
+
+  /* Support for Firefox. */
+  -moz-osx-font-smoothing: grayscale;
+
+  /* Support for IE. */
+  font-feature-settings: 'liga';
+}
```

Comment:
Blank line is missing.

A tips in order to prevent that.
<img width="580" alt="image" src="https://github.com/openfun/cunningham/assets/9265241/66b6a790-4516-4121-8f5a-aaf1e40112a7">

Reply:
Thanks for the tip!